src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -14977,35 +14944,32 @@ class FindRelatedBooksAsyncRequestAsyncPagedAllOdyssey
    /// <summary>
    /// Testing calling forms
    /// </summary>
-   public static void SampleFindRelatedBooks()
+   public static async Task SampleFindRelatedBooksAsync()
    {
-       public static async Task SampleFindRelatedBooksAsync()
+       LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+       FindRelatedBooksRequest request = new FindRelatedBooksRequest
        {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           FindRelatedBooksRequest request = new FindRelatedBooksRequest
-           {
-               BookNames = {
-                               new BookName("[SHELF_ID]", "[BOOK_ID]"),
-                           },
-               ShelvesAsShelfNames = {
-                                         new ShelfName("[SHELF_ID]"),
-                                     },
-           };
-           gax::PagedAsyncEnumerable<FindRelatedBooksResponse, BookName> response = libraryServiceClient.FindRelatedBooksAsync(request);
-           // Iterate over pages (of server-defined size), performing one RPC per page
-           foreach (BookName item in response)
-           {
-               // FIXME: inspect the results
-           }
-       }
-       // [END sample_core]
+           BookNames = {
```

Comment:
This isn't idiomatic C# formatting. Although I know it's sometimes hard to get this right using snippets; and this may well be being formatted by existing snippet code that's been incorrect for ages ;)

```
FindRelatedBooksRequest request = new FindRelatedBooksRequest
{
    BookNames =
    {
        new BookName(...),
    },
    ShelvesAsShelfNames =
    {
        new ShelfName(...),
    },
};
```

Reply:
Fighting snippets is sometimes like fighting css :(

Reply:
Sad but true!